core/templates/dev/head/forms/formBuilder.js

```diff
@@ -598,9 +603,27 @@ oppia.factory('rteHelperService', [
       // Returns a DOM node.
       createRteElement: function(componentDefn, customizationArgsDict) {
         var el = $('<img/>');
-        el.attr('src', componentDefn.iconDataUrl);
+        if (componentDefn.previewUrlTemplate) {
+          var interpolatedUrl = $interpolate(
+            componentDefn.previewUrlTemplate, false, null, true)(
+              angular.extend(customizationArgsDict,
+                {
+                  explorationId: explorationContextService.getExplorationId()
+                }
+              ));
+          if (typeof interpolatedUrl == 'undefined') {
```

Comment:
My initial thought was that some RTE component might have the empty string "" as an interpolated URL, but on hindsight perhaps that probably won't happen. I'll change it to `!interpolatedUrl`.